oak_restricted_kernel/src/payload.rs

```diff
@@ -115,6 +115,7 @@ pub fn run_payload(payload: &[u8]) -> ! {
     unsafe {
         asm! {
             "mov rsp, {}", // user stack
+            "and rsp, -10", // align stack to 16-byte boundary
```

Comment:
I think the context from the commit message may be useful in a comment here actually